stable/redis-ha/values.yaml

```diff
@@ -41,6 +41,7 @@ haproxy:
   ##
   service:
     type: ClusterIP
+    loadBalancerIP:
```

Comment:
I think this will have to be commented out or an empty string: `""`

Reply:
@ssalaues I tested this, should be fine. I'll check why the test is failing...